voluncheer/profiles/urls.py

```diff
@@ -1,12 +1,13 @@
 from django.urls import path
-from django.urls import re_path
```

Comment:
I thought you are supposed to use re_path, because otherwise you would get a warning for r"^profile/$", which is an outdated way? Maybe, my version was not correct when that warning showed up. So as long as you don't see any warning during migration and runserver, it should be good.

Reply:
Same as above. I did not implement this code so I am not sure what the person was intending.